### PR TITLE
Tdrd 500 update faq help

### PR DESCRIPTION
**Pull request by Tom-Hallett**

### CODE REVIEW

File: app/views/faq.scala.html [hunk 1]
```diff
@@ -354,8 +354,31 @@ <h3 class="govuk-heading-s">Can I include Redacted records?</h3>
...
+
+                <h3 class="govuk-heading-s">What is a Digital Transfer Advisor (DTA)?</h3>
+                <p class="govuk-body">
+                    A Digital Transfer Advisor (DTA) is your point of contact at The National Archives (TNA) who can provide you with advice and guidance with your selected born digital public record transfer to TNA.
+                </p>
+
+                <h3 class="govuk-heading-s">What is the purpose of a metadata review?</h3>
+                <p class="govuk-body">
+                    This step involves submitting your metadata for review by experts at The National Archives. The purpose is to ensure the metadata you have submitted has met TNA's standards for processing. Completing this accurately will ensure that we capture the information we need to add contextual value to the records. The review also ensures necessary access/closure conditions are adhered to.
+                    Records and metadata uploaded in TDR continue to be the responsibility of and owned by the transferring body. At the end of the preparation process, the transferring body agrees to transfer legal custody of the records and its metadata to The National Archives.
```
Comment (TomJKing) [inline app/views/faq.scala.html:368]: Should this be a new `<p/>` here?

Should `The National Archives` be `TNA`?

File: app/views/help.scala.html [hunk 2]
```diff
@@ -408,7 +409,79 @@ <h3 class="govuk-heading-s">Titles and descriptions that are sensitive to the pu
...
+                Once your metadata has been reviewed, you will receive a notification depending on one of the following outcomes:
+            </p>
+            <ol class="govuk-body">
+                <li>If issues are found with your metadata:
+                    <p class="govuk-body">
+                        You will receive two emails.
+                    </p>
+                    <p class="govuk-body">
+                        One from a Digital Transfer Advisor with recommendations for making the necessary amendments before submitting for another review.
+                        Another email will be sent with a link to your in progress transfer. This can also be accessed by signing into Transfer Digital Records, selecting the ‘View Transfers’ button and finding the transfer with the corresponding consignment reference. You will see that issues were found with your metadata.
```
Comment (TomJKing) [inline app/views/help.scala.html:452]: Should this be a new `<p/>`?

File: app/views/help.scala.html [hunk 3]
```diff
@@ -408,7 +409,79 @@ <h3 class="govuk-heading-s">Titles and descriptions that are sensitive to the pu
...
+                    <img src="@routes.Assets.versioned("images/standard-review-issues.png")" class="screenshots" alt="Issues found in metadata page" />
+                    <p class="govuk-body">
+                        Select ‘Continue’ to return to the ‘Record metadata’ page and address the issues with your metadata by selecting the record and updating the form.
+                    </p>
+                    <img src="@routes.Assets.versioned("images/standard-additional-metadata-short.png")" class="screenshots" alt="Additonal metadata page" />
+                    <p class="govuk-body">
+                        Once you have made the necessary changes, you will be prompted to repeat the steps described above to submit your metadata for review by the Digital Archiving team who will check it again and send you an email notification.
+                    </p>
+                </li>
+                <li>If issues are found with your metadata:
```
Comment (TomJKing) [inline app/views/help.scala.html:464]: Is this sentence correct? The steps outlined seem to be about if the metadata is OK?

**Review by TomJKing — COMMENTED** [review]
Looks good. Some minor comments. 

This should be reviewed by someone who knows what the content should actually be as well, eg Hari / Emma

**Review by annielh — APPROVED** [review]

**Pull request merged**